besu/src/main/java/org/hyperledger/besu/cli/options/SynchronizerOptions.java

```diff
@@ -293,12 +296,21 @@ public void parseBlockPropagationRange(final String arg) {
 
   @CommandLine.Option(
       names = SNAP_SERVER_ENABLED_FLAG,
-      hidden = true,
       paramLabel = "<Boolean>",
       arity = "0..1",
-      description = "Snap sync server enabled (default: ${DEFAULT-VALUE})")
+      description = "Enable snap sync server capability (default: ${DEFAULT-VALUE})")
   private Boolean snapsyncServerEnabled = SnapSyncConfiguration.DEFAULT_SNAP_SERVER_ENABLED;
 
+  @CommandLine.Option(
+      names = SNAP_SERVER_ENABLED_EXPERIMENTAL_FLAG,
+      hidden = true,
+      paramLabel = "<Boolean>",
+      arity = "0..1",
```

Comment:
Think you might need a `fallbackValue = true` so that it's possible to `--snapsync-server-enabled` without a value